pallets/did/src/lib.rs

```diff
@@ -355,6 +355,9 @@ pub mod pallet {
 		InvalidNonce,
 		/// The called extrinsic does not support DID authorisation.
 		UnsupportedDidAuthorizationCall,
+		/// The call had parameters that conflicted with each other.
```

Comment:
```suggestion
		/// The call had parameters that conflicted with each other
```

Reply:
Oh it goes on in the next line. didn't see that. 😂